news/_posts/releases/2022-01-26-prestashop-1-7-8-3-maintenance-release.md

```diff
@@ -0,0 +1,82 @@
+- [Fix recalculate shipping when the carrier of an order is changed](https://github.com/PrestaShop/PrestaShop/issues/25565)
+- [Fix the header search box oversized height for RTL languages](https://github.com/PrestaShop/PrestaShop/issues/26635)
+- [Using display hooks in order view produces error](https://github.com/PrestaShop/PrestaShop/issues/26471)
+- [Remove unnecessary scrollbar on customization fields on new order page](https://github.com/PrestaShop/PrestaShop/issues/26535)
+- [Remove lazy load for logo on Front Office](https://github.com/PrestaShop/PrestaShop/issues/26666)
+- [Replaces the new product flag in the product flags by new](https://github.com/PrestaShop/PrestaShop/issues/26640)
+- [Fix listing classes when using facetedsearch](https://github.com/PrestaShop/PrestaShop/issues/26820)
+- [Fix dropdown align on email theme page](https://github.com/PrestaShop/PrestaShop/issues/26604)
+- [Fix width and height value for product image in listing template](https://github.com/PrestaShop/PrestaShop/issues/25849)
+- [Remove header calculation and set toolbar to sticky](https://github.com/PrestaShop/PrestaShop/issues/26383)
+- [Fix a bad alignment of label in the email theme page](https://github.com/PrestaShop/PrestaShop/issues/26915)
+- [Fix the newsletter block overlap on customer reassurance  block](https://github.com/PrestaShop/PrestaShop/issues/26846)
+- [CI is broken on autoupgrade - PHPStan with 1.6.1.18](https://github.com/PrestaShop/PrestaShop/issues/26861)
+- [Remove duplication of cache folders in letting Symfony handle kernel reboot for tests](https://github.com/PrestaShop/PrestaShop/issues/26924)
+- [Fix untranslatable Tabs for shops upgraded to 1.7.8.0 (BC Break)](https://github.com/PrestaShop/PrestaShop/issues/26933)
+- [Fix menu and searchbar on tablet on classic (BC Break)](https://github.com/PrestaShop/PrestaShop/issues/26643)
+- [Allow vertical scroll on product images](https://github.com/PrestaShop/PrestaShop/issues/26597)
+- [Fix product width on featuredproducts](https://github.com/PrestaShop/PrestaShop/issues/26992)
+- [Allow all characters for Cart Rule & Order Cart Rule names](https://github.com/PrestaShop/PrestaShop/issues/25829)
+- [Add an option to regenerate RTL file during the upgrade](https://github.com/PrestaShop/PrestaShop/issues/27307)
+
+
+## Modules 
+
+- [Excludes vendor, lib, tests directories when scanning modules' directories to extract translation catalogue](https://github.com/PrestaShop/PrestaShop/issues/24987)
+- [Fix color picker bug when multiple colorpickers are present](https://github.com/PrestaShop/PrestaShop/issues/25013)
+- [Error with dynamical form rendering in 1.7.8.1](https://github.com/PrestaShop/PrestaShop/issues/26786)
+- [Fix product width on featured products](https://github.com/PrestaShop/PrestaShop/issues/26705)
+
+Read the [full changelog here](https://github.com/PrestaShop/PrestaShop/releases/tag/1.7.8.3).
```

Comment:
```suggestion
Read the [full changelog](https://github.com/PrestaShop/PrestaShop/releases/tag/1.7.8.3).
```

Reply:
or "Read the full changelog [here](...

Reply:
It's bad practice to use "here" or "click here" for links, both the original link or your 1st suggestions are ok